app.web-build.json

```diff
@@ -0,0 +1,15 @@
+{
+  "expo": {
+    "name": "react-native-elements-app",
+    "slug": "react-native-elements-app",
+    "description": "A demo app for React Native Elements UI Library",
+    "privacy": "public",
+    "sdkVersion": "34.0.0",
+    "platforms": ["web"],
+    "version": "1.1.0",
+    "primaryColor": "#cccccc",
+    "web": {
+      "publicPath": "/react-native-elements-app/"
+    }
```

Comment:
Strange expo behavior. We must set correct `publicPath` for production build (eg for gh-pages) but also we must keep `publicPath` with default value (`/`) for dev-env. I can't find any documentation to solve this problem, so separate `app.json` used for builds only

Reply:
Oh that is interesting